running/README.md

```diff
@@ -590,6 +590,49 @@ This can be achieved by setting the `max_jobs` property of a partition in the `s
 
 ```
 
+## Execution modes
+
+Reframe 2.5 permits you to define different *execution modes* of the framework.
+An execution mode is merely a set of predefined command-line options that will be passed to Reframe when this mode is invoked.
+You can define execution modes per system in the Reframe's configuration file.
+For example, one could define a global `maintenance` execution mode as follows in the `settings.py` file:
+
+```python
+site_configuration = ReadOnlyField({
+    'systems' : {
+        ...
+    },
+    'environments' : {
+        ...
+    },
+    'modes' : {
+        *' : {
+            'maintenance' : [
+                '--exec-policy=async',
+                '--output=$APPS/UES/$USER/regression/maintenance',
```

Comment:
Should we avoid to use out own paths here?
Why not:
`$APPS/regression/maintenance'`

The $USER/$USER is not needed. The same applies to `--logdir` and `--stage`
What do you think?